### PR TITLE
chore: added helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: helm/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  PUBLIC_MAP_SERVER_URL: {{ quote .Values.rasterCommon.serviceUrls.mapproxy }}
+  CATALOG_PUBLISHING_SERVICE_URL: {{ quote .Values.rasterCommon.serviceUrls.catalogManager }}
+  MAP_SERVER_CACHE_TYPE: {{ quote .Values.env.mapServerCacheType }}
+  DISPLAY_NAME_DIR: {{ quote .Values.env.displayNameDir }}
+  TILING_ZOOM_GROUPS: {{ quote .Values.env.tilingZoomGroups }}
+  HTTP_RETRY_ATTEMPTS: {{ quote .Values.env.httpRetry.attempts }}
+  HTTP_RETRY_DELAY: {{ quote .Values.env.httpRetry.delay }}
+  HTTP_RETRY_RESET_TIMEOUT: {{ quote .Values.env.httpRetry.resetTimeout }}
+  LINK_TEMPLATE: {{.Files.Get "config/linkTemplates.template" | quote }}
+  LAYER_SOURCE_DIR: "/layerSources"
```
Comment (cl-shaharshavit-rnd) [inline helm/templates/configmap.yaml]: move to deployment if used

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+env:
+  port: 80
+  targetPort: 8080
+  protocol: TCP
+  logLevel: info
+  logPrettyPrintEnabled: false
+  responseCompressionEnabled: true
+  requestPayloadLimit: 1mb
+  tilingZoomGroups: '0-5,6,7,8,9,10,11,12,13,14'
```
Comment (cl-shaharshavit-rnd) [inline helm/values.yaml]: update

**Review by cl-shaharshavit-rnd — APPROVED** [review]

File: helm/templates/configmap.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/configmap.yaml]: fixeda

File: helm/values.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml]: fixed

**Comment by AnguIar** [comment]
בס"ד
ישר כוח

**Pull request merged**